docs/getting-started/dev-quick-start/index.md

```diff
@@ -0,0 +1,342 @@
+# Developers Quick Start Guide
+
+## Introduction
+
+Use this guide to learn how to update a simple Howlite component. We recommend completing the general [quick start guide](/docs/getting-started/quick-start/index.md) first to get familiar with basic concepts.
+
+The scenario utilises a sample component library [Howlite](https://github.com/websight-io/howlite) and a demo site project _Luna_. For simplicity, we overrode the _Title_ component included in the [Howlite](https://github.com/websight-io/howlite) collection in advance and prepared the _Luna Title_ component inside the demo project. 
+
+Your task is to update the _Luna Title_ component and extend existing functionality and tests. See more details in the sections below.
+
+!!! info "Important notice"
+
+    All scripts presented below are designed for Linux-based platforms. If you are a Windows user, please install and configure [Windows Subsystem for Linux](https://learn.microsoft.com/en-us/windows/wsl/install) (WSL).
+
+## Part A: Prerequisites
+
+1. Install [AdoptOpenJDK 17](https://adoptium.net/) with 'x64/aarch64' architecture (on mac use `brew install openjdk@17`):
+``` script
+$ java --version
+openjdk 17.0.2 2022-01-18
+OpenJDK Runtime Environment Temurin-17.0.2+8 (build 17.0.2+8)
+OpenJDK 64-Bit Server VM Temurin-17.0.2+8 (build 17.0.2+8, mixed mode)
+```
+1. Install Maven 3.8.5+
+
+1. Install [Docker Desktop](https://www.docker.com/products/docker-desktop/).
+
+
+## Part B: Setup local environment
+
+### Download Maven configuration
```

Comment:
This step can be avoided if we define the `repository` in the `luna-parent` pom.xml:

```
  <repositories>
    <repository>
      <id>websight-repo</id>
      <url>https://repo.websight.io/maven/</url>
      <snapshots>
        <enabled>false</enabled>
      </snapshots>
    </repository>
  </repositories>
```

Reply:
@malaskowski great! let's do it. please update me when ready and I will remove this part.

Reply:
done